lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -170,6 +172,15 @@ local function submitFormForPlayer( data, endpoint, ply )
     )
 
     recordPlayerSubmission( ply )
+
+    for _,p in ipairs( player.GetHumans() ) do
+        if p:IsAdmin() then
```

Comment:
Might need to make this more generic and support stuff like moderators.

Reply:
Added moderator.